lib/html/pipeline/syntax_highlight_filter.rb

```diff
@@ -20,9 +20,10 @@ def call
           html = highlight_with_timeout_handling(text, lang)
           next if html.nil?
 
-          next unless (node = node.replace(html).first)
+          node.inner_html = html
           klass = node['class']
-          klass = [klass, "highlight-#{lang}"].compact.join ' '
+          scope = context[:scope] || "highlight-#{lang}"
```

Comment:
I would like to be able to scope my css, so I added this option.

For more info, https://github.com/jneen/rouge#scope-highlight